lib/coinbase/address.rb

```diff
@@ -194,5 +159,56 @@ def addresses_api
     def transfers_api
       @transfers_api ||= Coinbase::Client::TransfersApi.new(Coinbase.configuration.api_client)
     end
+
+    def destination_address_and_network(destination)
+      return [destination.default_address.id, destination.network_id] if destination.is_a?(Wallet)
+      return [destination.id, destination.network_id] if destination.is_a?(Address)
+
+      [destination, network_id]
```

Comment:
shouldnt this be an error?

Reply:
This is the base case when sending to a string address. Further validation e.g. that the address is formatted properly for the given network will be made server-side